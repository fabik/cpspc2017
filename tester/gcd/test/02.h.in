15
21 12 18 10 7 4 4 6 7 14 12 16 9 16 19
