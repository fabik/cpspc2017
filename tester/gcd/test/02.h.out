1
22
41
59
75
91
105
117
129
139
148
156
164
171
175
